### DISCUSSION TITLE
what is bypass load?

### DISCUSSION THREAD

**Discussion by blackwhitebear8** [wills106/homeassistant-solax-modbus#316] [Q&A]
What is bypass load sensor here? without the backup power connected it hovers between 30 to 50 watts. And when backup power is used that is counted ontop of those 30 to 50 watts.

**Comment by wills106** [comment]
I have moved this over to the discussions section as it's not really an issue, but a question.
It might be helpful if you let other people know which Inverter you have, at least the brand as there are 4 different brands of Inverters supported now.

It sounds like bypass load could potentially be an internal CT clamp measuring the base load of the Inverter?

**Reply by blackwhitebear8** [reply]
We have a solis inverter
